apis/v1beta1/opentelemetrycollector_types.go

```diff
@@ -94,6 +94,11 @@ type OpenTelemetryCollectorSpec struct {
 	// +required
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Config Config `json:"config"`
+	// ConfigVersions defines the number versions to keep for the collector config. Each config version is stored in a separate ConfigMap.
+	// Defaults to 3. The minimum value is 1.
+	// +optional
+	// +kubebuilder:default:=3
+	ConfigVersions *int32 `json:"configVersions,omitempty"`
```

Comment:
Does this need to be a pointer? It looks like it should always be set.

Reply:
I changed it to `int`, good point